apps/server/src/shared/repo/ltitool/ltitool.repo.spec.ts

```diff
@@ -75,7 +75,28 @@ describe('LtiTool Repo', () => {
 			expect(result.id).toEqual(entity.id);
 		});
 
-		it('should throw an Error if the scope mismatches the idtype', async () => {
+		it('should throw an error if the ltitool was not found', async () => {
+			const entity: LtiTool = ltiToolFactory.buildWithId();
+			await expect(repo.findByName(entity.name)).rejects.toThrow(NotFoundError);
+		});
+	});
+
+	describe('findByClientIdAndIsLocal', () => {
```

Comment:
maybe add a test that an ltitool without the islocal flag is not found

Reply:
Added the test